### ISSUE TITLE
doesn't work for ip addresses

### ISSUE THREAD

**Issue by maxlath** [craigmiskell/cookiemaster#8]
first of all, thanks for this awesome extension, very much what I was desperately looking for since the tragic events of the Great Firefox Extensions Purge! :D 

it looks like the extension fails to parse an IP addresses used in place of a domain name:

![cookie_ip](https://user-images.githubusercontent.com/1596934/71770153-cb591900-2f29-11ea-96f7-a1e8c9428b4d.png)

**Comment by craigmiskell** [comment]
Yeah, it's a bit ugly.  Noticed it myself on some LAN sites, and was wondering if anyone else would, and you're the first to say anything!

Fun fact: it's actually working (if you use another cookie editing extension you can see the cookie being set, or not, based whether cookiemaster allows/blocks for 0.0.1), but cookie-master doesn't show the cookie set/blocked icon.  I'm in the middle of a substantial rewrite right now (side effect of https://github.com/craigmiskell/cookiemaster/issues/7 and the rabbit hole I've gone down there), but I will look at making this behave better after that.

**Comment by craigmiskell** [comment]
PS: What/when was the  Great Firefox Extensions Purge?  I missed that.

**Comment by maxlath** [comment]
the Purge is the totally non-offical name I give to [when Firefox switched to WebExtensions](https://blog.mozilla.org/addons/2017/09/28/webextensions-in-firefox-57/) a few years ago, making many extensions obsolete, such as the now dead [CookieController](https://addons.mozilla.org/en/firefox/addon/cookie-controller/)

**Comment by craigmiskell** [comment]
Heh, lol, that's what triggered me to write CookieMaster
